package.json

```diff
@@ -66,8 +67,8 @@
     "eslint-config-oclif-typescript": "^1.0.2",
     "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-no-null": "^1.0.2",
-    "jest": "^26.6.3",
-    "ts-jest": "^26.5.6",
+    "jest": "^29.0.3",
+    "ts-jest": "^29.0.1",
```

Comment:
Why do we need this?

Reply:
The jest tests were failing in the workspace at version 26 and updating to 29 fixed it. Enough else has changed that I can try reverting that and see if the issue comes back.

Reply:
I tried and did not face any issue with 26.x versions